etc/kayobe/ansible/reboot.yml

```diff
@@ -2,6 +2,12 @@
 - name: Reboot the host
   hosts: seed-hypervisor:seed:overcloud:infra-vms
   serial: "{{ lookup('env', 'ANSIBLE_SERIAL') | default(1, true) }}"
+  vars:
+    gather_facts: false
+    reboot_with_bootstrap_user: false
+    ansible_user: "{{ bootstrap_user if reboot_with_bootstrap_user else kayobe_ansible_user }}"
+    ansible_python_interpreter: "{{ '/usr/bin/python3' if reboot_with_bootstrap_user else virtualenv_path + '/kayobe/bin/python' }}"
```

Comment:
This one could be overridden unconditionally.

Reply:
done